daily_milestone.go

```diff
@@ -1,29 +1,65 @@
 package main
 
-import "flag"
+import (
+	"flag"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var logger *log.Logger
+
+//LoggerSetup Initialization
+func LoggerSetup(info io.Writer) {
+	logger = log.New(info, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile)
+}
+
+func getProjectId(BaseUrl string, Token string, Projectname string, Namespace string) string {
+	headers := map[string]string{"PrivateToken": Token}
+	url := "https://" + BaseUrl + "/projects"
 
-func get_project_id(base_url string, token string, projectname string, namespace string) string {
-	headers := map[string]string{"PrivateToken": token}
-	url := "https://" + base_url + "/projects"
 	page := 1
+	strPage := strconv.Itoa(page)
+	s := []string{url, "?page=", strPage}
+	completeUrl := strings.Join(s, "")
 	for {
+		r, err := http.PostForm(completeUrl, url.Values())
 
 	}
 
 }
 
-//func get_milestones(base_url string, token string, project_id string)[]string{}
-//func create_milestone_data(advance int) []string{}
-//func create_milestones(base_url string, token string, project_id string, milestones string) string{}
+//func getMilestones(BaseUrl string, Token string, ProjectId string)[]string{}
+func createMilestonesData(Advance int) []string {
+	today := time.Now()
+	var list []int
+	for i := 0; i < Advance; i++ {
+		date := today + time.After(i*time.Minute)
```

Comment:
So the list of milestones is actually a list of tuples or a map.
[{dueDate: "2017-01-01",title: "2017-01-01"},{dueDate: "2017-01-02",title: "2017-01-02"}]
Depending on your viewpoint and the representation from the current Gitlab api this might be changed to a simple list of date strings.

Reply:
Oh I was definitely missing this. Thanks a lot.